data/near earth jobs.txt

```diff
@@ -0,0 +1,172 @@
+# Copyright (c) 2017 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Historical trip to <planet>"
```

Comment:
I think it'd be better to emphasize the field trip part, not the historical trip one.

Reply:
For what reason? The reason I emphasize the fact that it's a historical trip is because it's to the near Earth worlds, the oldest (known) colonies in human space, and even the planets of Sol, tying it more to the fact that it's a near Earth job instead of making it more general, in which case it would sound like it can offer anywhere.

Reply:
It sounds like someone's making the trip of his life that would eventually be recognized as a historical event, a paragraph for which is then placed in every history textbook.
That is, it sounds like it's about the actual Pilgrims coming to America, not a bunch of schoolchildren a thousand years into the future going on a field trip to explore their achievements.